apps/api/src/server/app.js

```diff
@@ -24,6 +25,10 @@ app.use(compression());
 app.use(morgan('combined'));
 app.use(helmet());
 
+if (process.env?.IS_CLIENT_CREDENTIAL_GRANT_FEATURE === 'true') {
```

Comment:
For consistency, I would use the `config` object, which contains all the env vars

Reply:
if we were to change this environment variable, we would need to reload the application to apply this middleware. It might make more sense to do this check inside the middleware itself, so that we would not need to restart the app

Reply:
ok